main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// A Recipe Struct allows you to insert recipe documents into your
+// collection
+
+type Recipe struct {
+	Name              string
+	Ingredients       []string
+	PrepTimeInMinutes int `json:"prepTimeInMinutes" bson:"prepTimeInMinutes"`
+}
+
+func main() {
+
+	// TODO:
+	// Replace the placeholder connection string below with your
+	// Altas cluster specifics. Be sure it includes
+	// a valid username and password! Note that in a production environment,
+	// you do not want to store your password in plain-text here.
+	var mongoUri = "<Your Atlas Connection String>"
+
+	// CONNECT TO YOUR ATLAS CLUSTER:
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(
+		mongoUri,
+	))
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = client.Ping(ctx, nil)
+
+	if err != nil {
+		log.Fatal("There was a problem connecting to your Atlas cluster. Check that the URI includes a validusername and password, and that your IP address has been whitelisted. Error: ", err)
+	}
+
+	fmt.Println("Connected to MongoDB!")
+
+	// Provide the name of the database and collection you want to use.
+	// If they don't already exist, the driver and Atlas will create them
+	// automatically when you first write data.
+	var dbName = "myDatabase"
+	var collectionName = "recipes"
+	collection := client.Database(dbName).Collection(collectionName)
+
+	/*      *** INSERT DOCUMENTS ***
+	 *
+	 * You can insert individual documents using collection.Insert().
+	 * In this example, we're going to create 4 documents and then
+	 * insert them all in one call with InsertMany().
+	 */
+
+	eloteIngredients := make([]string, 0)
```

Comment:
I don't have a strong opinion on whether to use a custom class for your documents, but you can make this array assignment code more concise by declaring it inline, e.g.:
```
eloteIngredients := [...]string{"corn", "mayonnaise", <and the rest here> }
```